components/Table/Table.js

```diff
@@ -85,7 +96,63 @@ export class Table {
       menuBtn.classList.add('menuBtn');
       menuBtn.textContent = 'more_vert';
 
-      actionsContainer.append(arrowBtn, menuBtn);
+      const dropDownMenu = document.createElement('div');
+      dropDownMenu.classList.add('drop-down__menu');
+      const menuItems = this.menuConfig.map((item) => {
+        const menuItem = document.createElement('a');
+        menuItem.classList.add('menu-item');
+        menuItem.textContent = item.label;
+
+        if (!col.sortable) {
+          if (item.key === 'sortbyasc') {
+            menuItem.style.display = 'none';
+          }
+          if (item.key === 'sortbydesc') {
+            menuItem.style.display = 'none';
+          }
+        }
+
+        menuItem.addEventListener('click', (e) => {
+          const dataKey = cell.getAttribute('data-key');
+
+          switch (item.key) {
+            case 'unsort':
+              this.sortCallback();
+              break;
+            case 'sortbyasc':
```

Comment:
The same as above - here. When we'll more readable name for this key and move it to the constant - it would be easier to support